backend/src/tasks/helpers/saveTrustymailResultsToDb.ts

```diff
@@ -0,0 +1,29 @@
+import { domain } from 'process';
+import { connectToDatabase, Domain } from '../../models';
+import * as fs from 'fs';
+
+// TODO: Find a way to only send one query to the db
```

Comment:
The way we'd only send one query is something similar to what we do with censysipv4 results and services (see https://github.com/cisagov/crossfeed/blob/b9fdf7f36d6dcfd54770ade62005a0df4a6e07ff/backend/src/tasks/censysIpv4.ts#L88-L90 and https://github.com/cisagov/crossfeed/blob/b9fdf7f36d6dcfd54770ade62005a0df4a6e07ff/backend/src/tasks/helpers/saveServicesToDb.ts).

We'd need to construct Domain objects and then modify saveDomainsToDb to ensure it's able to store the trustymailResults data.

Reply:
May not need to do this for this PR though.

Reply:
Have it working with one update query.